common/locals.js

```diff
@@ -183,15 +183,12 @@ module.exports = function (req, res, next) {
      * @param dateString
      */
     res.locals.formatCalendarTime = function (dateString) {
-        return (
-            moment(dateString)
-                .tz('Europe/London')
-                // @TODO: Handle this consistently across date formatters
```

Comment:
Double checked the original issue and this check is only needed on the `.calendar` method as it's the one place where date formats are generated (rather than explicitly defined).